test/metabase/sync_database/analyze_test.clj

```diff
@@ -72,3 +78,80 @@
 (expect false (values-are-valid-emails? [100]))
 (expect false (values-are-valid-emails? ["true"]))
 (expect false (values-are-valid-emails? ["false"]))
+
+;; Tests to avoid analyzing hidden tables
+(defn count-un-analyzed-fields [db_id table-name]
```

Comment:
I think `unanalyzed-fields-count` is a little clearer too (with `count` at the end) because otherwise it sounds like you're issuing some sort of command ("please count my fields") as opposed to just getting a count ("give me the count of fields")